src/modules/TodoItem.js

```diff
@@ -0,0 +1,9 @@
+class TodoItem {
+  constructor(description, completed, id) {
+    this.description = description;
+    this.completed = completed;
+    this.id = id;
```

Comment:
- [ ] Great job so far :100:  Kindly replace this `id` with `index` as it is part of the requirements for a `todo` object to have an `index` key :100: